src/devices/rosstech_dcu706.c

```diff
@@ -0,0 +1,149 @@
+/** @file
+    Rosstech Digital Control Unit DCU-706/Sundance
+
+    Copyright (C) 2023 suaveolent
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+#include "decoder.h"
+
+/**
```

Comment:
If this doc comment is not immediately before the function then use
`/** @fn int rosstech_dcu706_decode(r_device *decoder, bitbuffer_t *bitbuffer)`